webpack.config.js

```diff
@@ -0,0 +1,9 @@
+function buildConfig() {
+    const env = process.env.NODE_ENV;
+    if (env === 'dev' || env === 'prod') {
```

Comment:
Not an error, but you can use try catch also:
```js
try {
  return require(`./webpack.${env}.js`);
} catch(e) {
  console.log("Wrong webpack build parameter. Possible choices: 'dev' or 'prod'.")
}
```

Reply:
This function has been updated